charts/urban-os/values.yaml

```diff
@@ -75,6 +75,9 @@ raptor:
   fullnameOverride: raptor
   enabled: true
 
+sauron:
+  enabled: true
```

Comment:
Could we disable sauron by default? Just for a short while to see it working in the product environment, before it's a default for other deployments?

Reply:
Sure!